### ISSUE TITLE
Provide an alias for the negated boolean option

### ISSUE THREAD

**Issue by aks** [erikhuda/thor#592]
If a boolean option is to receive an automatically generated negative option, eg:
`option :unicode` also causes an option of `--no-unicode` to be allowed, then there should be a way to specify an alias to the negative switch.  Perhaps allow `:negative_aliases` option?

For example:
```
 option :unicode, aliases: '-U', negative_aliases: '-A', type: :boolean, default: false, desc: "Use Unicode for displays; if negated use ASCII"
```
or perhaps
```
 option :unicode, aliases: ['-U', '-A'], type: :boolean, default: false, desc: "Use Unicode for displays; if negated, use ASCII"
```
where the 2nd alias in the array argument to `aliases` is mapped to the negative switch?

**Comment by doudou** [comment]
Yep ... I currently work around it by providing definining a hidden option that has the alias, but that would definitely be better ;-)
